src/test/java/com/puppycrawl/tools/checkstyle/meta/MetadataGeneratorUtilTest.java

```diff
@@ -67,8 +73,30 @@ public void testMetadataFilesGenerationAllFiles() throws Exception {
                 .sorted()
                 .collect(Collectors.toCollection(LinkedHashSet::new));
         checkstyleModules.removeAll(modulesContainingNoMetadataFile);
-        assertEquals("Number of generated metadata files dont match with number of checkstyle "
-                        + "module", checkstyleModules, metaFiles);
+
+        Assertions.assertEquals(checkstyleModules, metaFiles,
+                "Number of generated metadata files dont match with number of checkstyle module");
+
+        final String[] expectedErrorMessages = {
+            "31: Description for 'AbstractSuper' is missing. [JavadocMetadataScraper]",
+            "41: Description for 'AbstractJavadoc' is missing. [JavadocMetadataScraper]",
+            "26: Description for 'AbstractAccessControlName' is missing. [JavadocMetadataScraper]",
+            "30: Description for 'AbstractName' is missing. [JavadocMetadataScraper]",
+            "43: Description for 'AbstractHeader' is missing. [JavadocMetadataScraper]",
+            "29: Description for 'AbstractParenPad' is missing. [JavadocMetadataScraper]",
+            "44: Description for 'AbstractClassCoupling' is missing. [JavadocMetadataScraper]",
+        };
+
```

Comment:
@rnveach This does not support localization yet, I can fix that but for that, I would have to extend `AbstractModuleTestSupport` but it's util so It won't be a good extend https://github.com/checkstyle/checkstyle/pull/10840#discussion_r735152673